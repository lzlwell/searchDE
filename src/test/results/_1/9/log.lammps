LAMMPS (30 Aug 2012)
units		metal
boundary	p p p
atom_style	atomic
read_data	struct.lammps
  orthogonal box = (0 0 0) to (17.1317 10 50)
  1 by 1 by 1 MPI processor grid
  300 atoms

pair_style      airebo 3.0
pair_coeff      * * CH.airebo C H

mass	1	12.0
mass	2	1.0

group		fixPosH id <> 1 120
120 atoms in group fixPosH
group		fixPosL id <> 121 239
119 atoms in group fixPosL
group		optPos  id <> 240 1000
61 atoms in group optPos
fix             0       fixPosH setforce NULL 0.0 NULL
fix             1       fixPosH aveforce 0.0 0.0 0.0
fix		2	fixPosL setforce 0.0 0.0 0.0
fix 		3	optPos	setforce NULL 0.0 NULL

neighbor        0.2 bin
neigh_modify	every 1 delay 0 check yes

timestep	0.002
thermo 		1
thermo_style    custom step  pe etotal lx ly lz 

dump		1 all xyz 10000000 dump.atom
dump_modify	1 format "%d  %20.15g  %20.15g  %20.15g" 
#fix 		8 all box/relax iso 0.0 
min_style	cg
minimize	1.0e-12 1.0e-12 10 10
Memory usage per processor = 3.2685 Mbytes
Step PotEng TotEng Lx Ly Lz 
       0   -715.89511   -715.89511      17.1317           10           50 
       1   -977.26873   -977.26873      17.1317           10           50 
       2   -1052.7221   -1052.7221      17.1317           10           50 
       3   -1092.7067   -1092.7067      17.1317           10           50 
       4   -1110.7179   -1110.7179      17.1317           10           50 
       5   -1125.7338   -1125.7338      17.1317           10           50 
       6   -1143.5452   -1143.5452      17.1317           10           50 
       7   -1150.8366   -1150.8366      17.1317           10           50 
       8   -1155.6824   -1155.6824      17.1317           10           50 
       9   -1160.0197   -1160.0197      17.1317           10           50 
      10   -1162.5696   -1162.5696      17.1317           10           50 
Loop time of 0.37295 on 1 procs for 10 steps with 300 atoms

Minimization stats:
  Stopping criterion = max force evaluations
  Energy initial, next-to-last, final = 
        -715.895113239      -1160.0196743     -1162.56958692
  Force two-norm initial, final = 1857.39 37.1335
  Force max component initial, final = 935.287 19.7686
  Final line search alpha, max atom move = 0.00619286 0.122424
  Iterations, force evaluations = 10 10

Pair  time (%) = 0.137694 (36.9201)
Neigh time (%) = 0.231634 (62.1087)
Comm  time (%) = 0.00149941 (0.402042)
Outpt time (%) = 0.000158072 (0.0423841)
Other time (%) = 0.00196457 (0.526765)

Nlocal:    300 ave 300 max 300 min
Histogram: 1 0 0 0 0 0 0 0 0 0
Nghost:    2499 ave 2499 max 2499 min
Histogram: 1 0 0 0 0 0 0 0 0 0
Neighs:    0 ave 0 max 0 min
Histogram: 1 0 0 0 0 0 0 0 0 0
FullNghs:  45056 ave 45056 max 45056 min
Histogram: 1 0 0 0 0 0 0 0 0 0

Total # of neighbors = 45056
Ave neighs/atom = 150.187
Neighbor list builds = 10
Dangerous builds = 10
min_modify line quadratic

clear
