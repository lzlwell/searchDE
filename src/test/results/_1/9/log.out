LAMMPS (30 Aug 2012)
Reading data file ...
  orthogonal box = (0 0 0) to (17.1317 10 50)
  1 by 1 by 1 MPI processor grid
  300 atoms
120 atoms in group fixPosH
119 atoms in group fixPosL
61 atoms in group optPos
Setting up minimization ...
Memory usage per processor = 3.2685 Mbytes
Step PotEng TotEng Lx Ly Lz 
       0   -715.89511   -715.89511      17.1317           10           50 
       1   -977.26873   -977.26873      17.1317           10           50 
       2   -1052.7221   -1052.7221      17.1317           10           50 
       3   -1092.7067   -1092.7067      17.1317           10           50 
       4   -1110.7179   -1110.7179      17.1317           10           50 
       5   -1125.7338   -1125.7338      17.1317           10           50 
       6   -1143.5452   -1143.5452      17.1317           10           50 
       7   -1150.8366   -1150.8366      17.1317           10           50 
       8   -1155.6824   -1155.6824      17.1317           10           50 
       9   -1160.0197   -1160.0197      17.1317           10           50 
      10   -1162.5696   -1162.5696      17.1317           10           50 
Loop time of 0.37295 on 1 procs for 10 steps with 300 atoms

Minimization stats:
  Stopping criterion = max force evaluations
  Energy initial, next-to-last, final = 
        -715.895113239      -1160.0196743     -1162.56958692
  Force two-norm initial, final = 1857.39 37.1335
  Force max component initial, final = 935.287 19.7686
  Final line search alpha, max atom move = 0.00619286 0.122424
  Iterations, force evaluations = 10 10

Pair  time (%) = 0.137694 (36.9201)
Neigh time (%) = 0.231634 (62.1087)
Comm  time (%) = 0.00149941 (0.402042)
Outpt time (%) = 0.000158072 (0.0423841)
Other time (%) = 0.00196457 (0.526765)

Nlocal:    300 ave 300 max 300 min
Histogram: 1 0 0 0 0 0 0 0 0 0
Nghost:    2499 ave 2499 max 2499 min
Histogram: 1 0 0 0 0 0 0 0 0 0
Neighs:    0 ave 0 max 0 min
Histogram: 1 0 0 0 0 0 0 0 0 0
FullNghs:  45056 ave 45056 max 45056 min
Histogram: 1 0 0 0 0 0 0 0 0 0

Total # of neighbors = 45056
Ave neighs/atom = 150.187
Neighbor list builds = 10
Dangerous builds = 10
